tst/testinstall/mapping.tst

```diff
@@ -403,5 +403,16 @@ true
 gap> IsInjective(res);        
 true
 
+# printing of identity mapping in direct product element (PR #3753) 
+gap> hom := GroupHomomorphismByImages(g,g,[(1,2),(3,4)],[(3,4),(1,2)]);
+[ (1,2), (3,4) ] -> [ (3,4), (1,2) ]
+gap> aut := Group(hom);;
+gap> dp := DirectProduct(aut,aut);;
+gap> GeneratorsOfGroup(dp);
+[ DirectProductElement( [ [ (1,2), (3,4) ] -> [ (3,4), (1,2) ], 
+      IdentityMapping( Group( [ (1,2), (3,4) ] ) ) ] ), 
+  DirectProductElement( [ IdentityMapping( Group( [ (1,2), (3,4) ] ) ), 
+      [ (1,2), (3,4) ] -> [ (3,4), (1,2) ] ] ) ]
```

Comment:
That's good, but I think a much more direct test would be better (though this one can be left in as well). I.e., just something like this:
```
gap> String(IdentityMapping(SymmetricGroup(3)));
"IdentityMapping( Sym( [ 1 .. 3 ] ) )"
```
(That said, I am also fine with merging this PR as-is).